contribute.md

```diff
@@ -65,13 +65,13 @@ order. Link to the appropriate graphic rather than duplicating it.
 To reference image files in GitHub, use the following markup:
 
     {% figure [caption:"<caption information>"] [class:"image bg-light border"] %}
-    ![]({{site.url}}<relative-path>){:width="100%"}
+    ![]({{site.url}}/assets/images/<relative-path>){:width="100%"}
     {% endfigure %}
 
 For example, if you are working on the Modal page:
 
     {% figure [caption:"Diagram showing modal operation"] [class:"image bg-light border"] %}
-    ![]({{site.url}}components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
+    ![]({{site.url}}/assets/images/components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
```

Comment:
Same here:
```
![]({{site.url}}assets/images/components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
```

Reply:
Based on my testing, it does not work without that slash.

Reply:
Corrected